text/utilities/0004-source-removal-buildpack.md

```diff
@@ -33,7 +33,16 @@ ecosystem and as needed by particular users.
   benefit from such functionality.
 
 ## Implementation
+The Rust Buildpacks will be moved into the `paketo-buildpacks` Github
```

Comment:
A typo, I think?

Reply:
I think that is a typo I have already remedied maybe you are looking at an old commit